### ISSUE TITLE
[Anaconda] This Package does not update the anaconda navigator?

### ISSUE THREAD

**Issue by BrandonKMLee** [teknowledgist/Chocolatey-packages#64]
Checking https://github.com/teknowledgist/Chocolatey-packages/tree/master/automatic/anaconda3 and how the distribution is separate from the navigator.
https://docs.anaconda.com/anaconda/navigator/update-navigator/
`conda deactivate`
`conda update anaconda-navigator`

**Comment by teknowledgist** [comment]
I'm willing to update navigator upon install of Anaconda3 if there is a way to do so from PowerShell.  I am not a Python/Anaconda user (just a sysAdmin who needs to deploy it), so I don't know what options there are for scripting updates like this.  Keep in  mind that this needs to work across a wide range of PowerShell versions.

I'd be willing to create a separate, "anaconda3-navigator" package with a dependency on "anaconda3", but I would need to have a way to download/install *just* navigator and identify what version it is.  

Any ideas about how/where I can do either of those?

**Comment by BrandonKMLee** [comment]
It seemend to be something like
```
conda deactivate
conda update anaconda-navigator
```
Used by:
- https://appdividend.com/2020/05/12/how-to-update-anaconda-upgrade-anaconda-navigator/
- https://medium.com/@woraperth/python-how-to-update-anaconda-navigator-with-command-line-97cbb26f77de
- https://github.com/ContinuumIO/anaconda-issues/issues/12135
- https://www.reddit.com/r/learnpython/comments/kcc6gq/anaconda_im_new_and_i_cant_update_the_anaconda/

**Comment by teknowledgist** [comment]
Yes, it seems to be simple *from within the Anaconda Prompt*.  I need to be able to do it from a PowerShell script.  

At least one of those links makes it appear to have user prompts which I can't do either.  

Finally, this would only run one time when the Chocolatey package is installed.  Without a separate anaconda-navigator package, I know of no way to auto-update the navigator component of Anaconda via Chocolatey.
